image/store/distribution.go

```
@@ -41,7 +41,7 @@ func (fs *filesystem) LoadDistributionMetadata(layerID LayerID) (map[string]dige
 	)
 	distributionMetadataFile, err := os.Open(filepath.Join(layerDBPath, "distribution_layer_digest"))
 	if err != nil {
-		return res, nil
+		return res, err
```

ignore this error, build will failed if return error here.